app/controllers/agenda/agendaitems/agendaitem/index.js

```diff
@@ -25,6 +26,10 @@ export default class IndexAgendaitemAgendaitemsAgendaController extends Controll
 
   @tracked isEditingAgendaItemTitles = false;
 
+  get isFinalMeeting() {
```

Comment:
`isFinalAgenda` instead of `isFinalMeeting`

Reply:
Actually, `isClosedMeeting` would be even better semantically.